src/main/java/com/gw/tools/ProcessTool.java

```diff
@@ -737,7 +737,7 @@ public String all_history(String pid, boolean ignore_skipped) {
 	
 	public String all_history(String pid) {
 
-		return all_history(pid, true);
```

Comment:
this should be dynamic. Passing true always will show skipped process always.
As of now it is hardcoded to be false. So lets make sure we only show skipped when we click a button or use a certain filter and not show it always.